asmcli/commands/install.sh

```diff
@@ -206,3 +206,68 @@ install_control_plane() {
     install_canonical_controller
   fi
 }
+
+### Installation functions ###
+configure_package() {
```

Comment:
`configure_package` is also used to print config. I'm not sure exactly the best place to put it--maybe lib/config.sh or something?

In general, the stuff in "commands" I'd like to be as tightly scoped to just that command as possible.

Reply:
Put it to lib/config.sh